README.md

```diff
@@ -44,17 +44,25 @@ Other API features are documented in the [API Reference](https://conduce-conduce
 
 ## Getting started
 
-The Conduce command line interface provides users with utilities for exercising common API commands, and textual feedback for inspecting and debugging.  If you want to use the CLI without any setup you may fully specify your requests on the command line.
+The Conduce command line interface provides users with utilities for exercising common API commands, and textual feedback for inspecting and debugging. Features include:
+
+- Creating API keys
+- Adding and removing datasets
+- Creating a dataset from a CSV or JSON file
+- Listing datasets and other resources
+
+```
+All CLI functionality is available through `conduce/api.py` and all API functionality should be usable with or without a `.conduceconfig` file. If you want to use the CLI without any setup you may fully specify your requests on the command line.
 
 As an example, you may list existing datasets with the following command:
 
 ```
-conduce-api list datasets --host=app.conduce.com --user=email@example.com
+conduce-api list datasets --host=app.conduce.com --user=email@example.com --api-key=<your api key>
```

Comment:
I think this needs to change back.  If the user thinks they need to specify an API key here that will slow them down.  Remember, the user doesn't know about protools and they've never done anything with Conduce, they don't have an API key.  Username/password authentication is all that is available.

Reply:
you're right. I made this back before I had the password prompt working